src/libsyncengine/update_detection/file_system_observer/localfilesystemobserverworker.cpp

```diff
@@ -700,13 +716,13 @@ ExitInfo LocalFileSystemObserverWorker::exploreDir(const SyncPath &absoluteParen
                                                                                            isWarning, isExcluded, ioError);
                 if (!success) {
                     LOGW_SYNCPAL_DEBUG(_logger, L"Error in ExclusionTemplateCache::isExcluded: "
-                                                        << Utility::formatIoError(absolutePath, ioError).c_str());
+                                                        << Utility::formatIoError(absolutePath, ioError));
                     dirIt.disableRecursionPending();
                     continue;
                 }
                 if (isExcluded) {
-                    LOGW_SYNCPAL_INFO(_logger, L"Item: " << Utility::formatSyncPath(absolutePath).c_str()
-                                                         << L" rejected because it's excluded");
+                    LOGW_SYNCPAL_INFO(_logger,
+                                      L"Item: " << Utility::formatSyncPath(absolutePath) << L" rejected because it's excluded");
```

Comment:
```suggestion
                                      L"Item: " << Utility::formatSyncPath(absolutePath) << L" rejected because it is excluded");
```

Reply:
Why not...

Reply:
Same: avoid potential visual conflicts with paths or names delimiters.

Reply:
Done